docs/src/architecture/10_quality/testplans/flutter_cardano_integration.md

```diff
@@ -0,0 +1,92 @@
+<!--
+Based on: https://books.google.it/books?id=vHlTOVTKHeUC&hl=it&source=gbs_navlinks_s
+          https://testing.googleblog.com/2016/06/the-inquiry-method-for-test-planning.html
+          https://testing.googleblog.com/2011/09/10-minute-test-plan.html
+-->
+
+# [Flutter Cardano Integration](https://github.com/input-output-hk/catalyst-voices/issues/481)
+
+* [General informations](#general-informations)
+  * [Abstract](#abstract)
+  * [Stakeholders](#stakeholders)
+  * [Requirements](#requirements)
+  * [Tools](#tools)
+* [ACC framework](#acc-framework)
+  * [Attributes](#attributes)
+  * [Components](#components)
+  * [Capabilities](#capabilities)
+* [Integration tests](#integration-tests)
+  * [Integration tests strategy](#integration-tests-strategy)
+  * [Integration test cases](#integration-test-cases)
+
+
+## General Informations
+
+### Abstract
+Catalyst Cardano is a web application that facilitate interaction with the Cardano blockchain, incorporating CIP-30 (Wallet DAppBridge) and CIP-95 (Message Signing) specifications. It requires connection to the wallet third party extensions (e.g Eternl) to be able to perform actions like retrieving wallet details, signing data/transactions and submiting transactions.
+
+### Stakeholders
+
+| Role/Name       | Contact        | Approval       |
+|-----------------|----------------|----------------|
+| *Eng-Manager*   |     Sasha      |                |
+| *Developers*    |     Dominik    |                |
+| *Testers*       |     Duy        |                |
+
+### Requirements
+
+[EPIC](https://github.com/input-output-hk/catalyst-voices/issues/399)  
+[CIP-30](https://cips.cardano.org/cip/CIP-30)  
+[CIP-95](https://cips.cardano.org/cip/CIP-95)  
+Testing will be focused only on web applications for the time being.  
+Focus needs to be on compliance with CIP-30 and CIP-95 specs, reliability and security especially when handling wallet interactions and message signing. UI testing is out of scope, the web application will be used only for setup purpose.  
+
+### Acceptance criteria
+
+- The package must comply with CIP-30 and CIP-95 specifications, focusing on the web.  
+- High unit test coverage to ensure reliability and security.  
+- Clear and comprehensive documentation for developers.  
+- An example web application that effectively demonstrates the package's functionalities.  
+
+### Risks
+
+Documentation is not complete yet, integration dart package might be not enough  
+
+### Tools
+
+The main tool we will use is dart integration test package. We still need to investigate if all the testing needed can be archived only with dart or if we need to use playwright for the web UI part  
+
+## ACC framework
+
+> :memo: **Note:** Use the [ACC framework](https://testing.googleblog.com/2011/09/10-minute-test-plan.html)
+> to help you define the test cases
+
+### Attributes
+
+Secure, complaint with CIP 30 and 36, reliable    
```

Comment:
CIP-36 is something that we will aim to achieve using catalyst cardano but the library mainly targets CIP-30 and CIP-95